js/qm-ionic-app.js

```diff
@@ -4,9 +4,10 @@ jQuery(document).ready(function () {
     var showHideButton = jQuery('#qm-ionic-app-show-hide');
     var iFrame = jQuery('#ionic-app-holder iframe');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['iFrame' is defined but never used.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1151086236/issues/source?bid=2836188&fileBranchId=2912624#l5)